core/templates/dev/head/pages/Base.ts

```diff
@@ -123,7 +123,6 @@ angular.module('oppia').controller('Base', [
       // TODO(sll): use 'touchstart' for mobile.
       $document.on('click', function() {
         SidebarStatusService.onDocumentClick();
-        $scope.$apply();
```

Comment:
This is related to the sidebar closing functionality on clicking anywhere on the screen (on mobile devices). We don't need an $apply here. I've tested this change in responsive mode on Firefox and Chrome.